pkgs/games/dwarf-fortress/default.nix

```diff
@@ -35,6 +47,13 @@ let
   dfHackWorksWithCurrentVersion = true;
   dfHackEnabled = dfHackWorksWithCurrentVersion && enableDFHack;
 
+  themesThatWorkWithCurrentVersion = [
```

Comment:
That's way more work for the maintainer - and the list might be outdated when the themes update. I would drop this completely and just let the user use any theme.